src/components/EditorSource.vue

```diff
@@ -67,10 +69,11 @@ export default {
     /**
      * Value to show in the content type field
      */
-    contentType: {
-      required: true,
-      type: String,
-    },
+    // FIXME: Support other content types and protocol options
+    // contentType: {
+    //   required: true,
+    //   type: String,
+    // },
```

Comment:
`contentType` should be a prop in spite of we only allow one value. Doing that, when we support more contentTypes, we only have to remove the `disabled option`